src/resources/views/crud/columns/color.blade.php

```diff
@@ -0,0 +1,28 @@
+{{-- regular object attribute --}}
+@php
```

Comment:
Probably this columns could have a `showColorHex => true/false` to display the Hex as text to copy ? 
![image](https://user-images.githubusercontent.com/7188159/198282070-71789498-d9ed-428b-bc31-51735167b8b2.png)

Reply:
applied `showColorHex` attribute as discussed (default value is false)